src/components/NavDesktop.js

```diff
@@ -8,6 +8,22 @@ import AccountNav from './AccountNav';
 import AsylumConnectButton from './AsylumConnectButton';
 import FavoritesLink from './FavoritesLink';
 import Language from './Language';
+import '../App.scss'
```

Comment:
You can remove this import once the styles from that file are moved here. Also, I dont believe you have to keep importing the `App.scss` fine since its already imported in `App.js` and thus included throughout the app